File: problem-solving/algorithms/java/HackerrankPsAlgorithms0001.java
import java.util.Scanner;

public class HackerrankPsAlgorithms0001 {

	// Solve Me First
	// https://www.hackerrank.com/challenges/solve-me-first/problem?isFullScreen=true

	// --- You need to fill this method ---
	static int solveMeFirst(int a, int b) {
		return a + b;
	}

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int a;
		a = in.nextInt();
		int b;
		b = in.nextInt();
		int sum;
		sum = solveMeFirst(a, b);
		System.out.println(sum);
	}

}

File: problem-solving/algorithms/dotnet/HackerrankPsAlgorithms0001.cs
using System;

class Solution
{
	// Solve Me First
	// https://www.hackerrank.com/challenges/solve-me-first/problem?isFullScreen=true

	static int solveMeFirst(int a, int b)
	{
		// --- You need to fill this method ---
		return a + b;
	}

	public static void Main(String[] args)
	{
		int val1 = Convert.ToInt32(Console.ReadLine());
		int val2 = Convert.ToInt32(Console.ReadLine());
		int sum = solveMeFirst(val1, val2);
		Console.WriteLine(sum);
	}

}
